frontend/src/App.svelte

```diff
@@ -1,19 +1,39 @@
 <script>
+    import { onMount } from "svelte";
     import { Router, Route } from "svelte-routing"; 
     import Sidebar from "./Components/Sidebar.svelte";
+    import Navbar from "./Components/Navbar.svelte"; // Import the responsive navbar component
     import Home from "./Pages/Home.svelte";
     import EventDetail from "./Pages/Events/EventDetail.svelte";
     import EventCreate from "./Pages/Events/EventCreate.svelte";
     import Profile from "./Pages/Profile.svelte";
+    import EventCard from "./Components/Events/EventCard.svelte";
+
+    let isSmallScreen = false;
+
+    // Check window size on mount and set the isSmallScreen variable
+    onMount(() => {
+        isSmallScreen = window.innerWidth <= 768;
+    });
+
+    // Add a resize event listener to dynamically update isSmallScreen variable
+    window.addEventListener("resize", () => {
+        isSmallScreen = window.innerWidth <= 768;
+    });
 </script>
 
+
 <Router>
-    <!--<Navbar />-->
     <div class="app">
-        <Sidebar />
+        {#if isSmallScreen}
+            <Navbar /> <!-- Show the responsive navbar on small screens -->
+        {:else}
+            <Sidebar /> <!-- Show the sidebar on larger screens -->
+        {/if}
         <div class="main-content">
             <Route component={Home} /> <!--Default route to home-->
             <Route path="/profile" component={Profile} />
+            <Route path="/eventcards" component={EventCard}/>
```

Comment:
Since we aren't using this anywhere could we get rid of this page?

Reply:
Yes